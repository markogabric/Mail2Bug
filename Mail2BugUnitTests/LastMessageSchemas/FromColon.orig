<html>
<body>
<div class="wrapppingBothMessageAndReply">
    <p class="customStyling">This is random text with some custom styling and outlook-generated elements.<o:p></o:p></p>
    <div>
        <div style="border:none;border-top:solid">
            <p class="customStyling"><b>From:</b> someAddress;<br><b>Subject:</b> RE: Build error<o:p></o:p>
            </p>
        </div>
    </div>
    <p class="customStyling">
        <o:p>&nbsp;</o:p>
    </p>
    <p class="customStyling">text of the reply
    </p>
</div>
<div>Something after the containing div</div>
</body>
</html>