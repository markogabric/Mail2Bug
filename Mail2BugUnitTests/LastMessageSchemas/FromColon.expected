<html><head></head><body>
<div class="wrapppingBothMessageAndReply">
    <p class="customStyling">This is random text with some custom styling and outlook-generated elements.<o:p></o:p></p>
    <div>
        <div style="border:none;border-top:solid">
            <p class="customStyling"><b></b></p></div></div></div></body></html>